.github/CODEOWNERS

```diff
@@ -387,6 +387,9 @@ sdk/ai/ai-inference-rest @glharper @dargilco
 # PRLabel: %Mgmt
 /sdk/cognitiveservices/arm-cognitiveservices/ @qiaozha @MaryGao
 
+# PRLabel: %Mgmt
+/sdk/confidentialledger/arm-confidentialledger/ @qiaozha @MaryGao
```

Comment:
@jeremymeng FYI - I directly forced merging this pr even I have codeowners change. Let me know if any concern!

![image](https://github.com/user-attachments/assets/6d1d4798-64b7-4014-b3b0-d022ced6997b)

Reply:
@MaryGao looks good!  The protection on CODEOWNERS is to prevent outside attacks